ex/ThirdPartyDemos/martin-doyle/01 - Stand Alone ORM/source/client.pas

```diff
@@ -0,0 +1,14 @@
+unit client;
+
+interface
+uses
+  {$I mormot.uses.inc}
```

Comment:
mormot.uses.inc should not be part of the units uses clause - just the main program / .dpr.

What is missing on the units is {$I mormot.defines.inc} at the top of every unit, e.g. between "interface" and "uses".

Reply:
OK. Finally I understood how to use it.